.wp-env.json

```diff
@@ -8,7 +8,7 @@
 				"wp-content/plugins/gutenberg": ".",
 				"wp-content/mu-plugins": "./packages/e2e-tests/mu-plugins",
 				"wp-content/plugins/gutenberg-test-plugins": "./packages/e2e-tests/plugins",
-				"wp-content/themes/gutenberg-test-themes": "./packages/e2e-tests/themes"
+				"wp-content/themes": "./test/gutenberg-test-themes"
```

Comment:
We'll need to `encodeURIComponent` in `activateTheme()` to support theme names like gutenberg-test-themes/style-variations